Gemfile

```
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-RAILS_GEMS_VERSION = "~> 7.2.0".freeze
+RAILS_GEMS_VERSION = "~> 7.2.1".freeze
```

The flexible range here seems a little curious. I assume you don't actually want to have any variance between the Rails gems use - presumably Rails mostly blocks that anyway with each gem having a fixed dependency on active support. 

We have a convention to use the fixed Rails versions in apps as otherwise you run the risk of Rails version being downgraded by Gems that depend on Rails: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#gemfile-organisation

```suggestion
RAILS_GEMS_VERSION = "7.2.1".freeze
```

---

Great spot - have pinned the version properly 🙏